CHANGELOG.md

```diff
@@ -5,20 +5,65 @@ All notable changes to Zebra are documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org).
 
 
-## Next Release (Draft)
+## [Zebra 1.0.0-beta.13](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-beta.13) - 2022-07-11
 
-This release improves Zebra's sync and verification performance under heavy load.
-(TODO - complete the summary.)
+This release fixes multiple bugs in proof and signature verification, which were causing big performance issues near the blockchain tip.
```

Comment:
Not true yet.

Reply:
I think it's ok to just get CI passing for this release, and do more performance improvements in the next release.

What do you think?

Reply:
Yes i am fine with that, if we can fix the CI, then we just need to change the description and add the items that we merged and are not in the changelog yet.

Reply:
Also, we should add what it is taking the most time to the known issues.

Reply:
Here are some of the known issues and ongoing investigations:
- #4824
- #4822
- #4794
- #4784

RPCs might also be slower than they used to be, we need to check:
- #4831

And here are some of the ongoing investigations:
- #4823
- #4747
- #4583

Reply:
I added this list at https://github.com/ZcashFoundation/zebra/pull/4765/commits/71748e6a9572bbb93b563eaf5c7338d27cd904ba